integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/IdentityProvidersResource.java

```diff
@@ -43,6 +44,16 @@ public interface IdentityProvidersResource {
     @Produces(MediaType.APPLICATION_JSON)
     List<IdentityProviderRepresentation> findAll();
 
+    @GET
+    @Path("instances")
+    @Produces(MediaType.APPLICATION_JSON)
+    List<IdentityProviderRepresentation> find(@QueryParam("search") String search, @QueryParam("first") Integer firstResult, @QueryParam("max") Integer maxResults);
+
+    @GET
+    @Path("instances/count")
```

Comment:
Why do we have a separate `count` endpoint? Under what circumstances should we be using this?